invenio_theme_tugraz/config.py

```diff
@@ -90,7 +91,7 @@
 # See https://invenio-rdm-records.readthedocs.io/en/latest/configuration.html
 # Uncomment below to override records landingpage.
 # from invenio_rdm_records.config import RECORDS_UI_ENDPOINTS
```

Comment:
Add the import on top of the file.

```suggestion
from invenio_rdm_records.config import RECORDS_UI_ENDPOINTS
```

Reply:
should i use the import from invenio_rdm_records instead of invenio_app_rdm. i thought that the landing page moved to invenio_app_rdm

Reply:
@utnapischtim if it moved then use the new location.